runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -310,10 +310,11 @@ void setupSwaps()
 
 		swap("view offer", "abort offer", () -> shiftModifier() && config.swapGEAbort());
 
-		Arrays.asList(
-			"honest jimmy", "bert the sandman", "advisor ghrim", "dark mage", "lanthus", "turael", "mazchna", "vannaka",
-			"chaeldar", "nieve", "steve", "duradel", "krystilia", "konar", "murphy", "cyrisus", "smoggy", "ginea", "watson",
-			"barbarian guard", "amy", "random"
+		Arrays.asList( //newlines in accordance to UI-layout
```

Comment:
?

Reply:
Linebreaks placed to correspond to the layout of the ingame UI

Reply:
Ah, I see. That doesn't need a comment here, though the commit message should probably mention that they've been reformatted here.